src/main/java/com/spikes2212/command/AddressableLEDWrapper.java

```diff
@@ -0,0 +1,123 @@
+package com.spikes2212.command;
+
+import edu.wpi.first.wpilibj.AddressableLED;
+import edu.wpi.first.wpilibj.AddressableLEDBuffer;
+
+import java.awt.*;
+
+/**
+ * A class that handles the communication between an LED strip and the code.
+ *
+ * @author Camellia Lami
+ * @see DashboardedSubsystem
+ */
+public class AddressableLEDWrapper {
+
+    /**
+     * {@link AddressableLED} - The LED strip itself.
```

Comment:
Each field should have its own documentation (javadoc comments are tied to the things they describe)